src/library-authoring/section-subsections/LibraryContainerChildren.tsx

```diff
@@ -61,8 +61,16 @@ const ContainerRow = ({ containerKey, container, readOnly }: ContainerRowProps)
     <>
       {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
       <div
-        // Prevent parent card from being clicked.
-        onClick={(e) => e.stopPropagation()}
+        onClick={(e) => {
+          // Wrap the actions in a div to prevent the card from being clicked when the actions are clicked.
+          const target = e.target as HTMLElement;
+          const isDropdownToggle = target.closest('[data-testid="dropdown"], [data-testid="container-card-menu-toggle"], .pgn__dropdown-toggle-iconbutton');
```

Comment:
I'm not comfortable with `testid` attributes being used in running code -- and they're even a last resort when identifying elements in tests. Can we use classes here instead?

Reply:
And is there any way to avoid duplicating this logic every time we need it? E.g using a utility function, or at least only declaring the function once per file?